### PR TITLE
Add warning about config changes.

### PR DESCRIPTION
**Pull request by fabxc**
I don't think we have to explain the specific changes in this message.
Did you have anything more elaborate in mind, @juliusv.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -92,7 +92,8 @@ type prometheus struct {
 func NewPrometheus() *prometheus {
 	conf, err := config.LoadFromFile(*configFile)
 	if err != nil {
-		glog.Errorf("Couldn't load configuration (-config.file=%s): %v\n", *configFile, err)
+		glog.Errorf("Couldn't load configuration (-config.file=%s): %v", *configFile, err)
+		glog.Errorf("Note: The configuration format has changed with version 0.14, please check the documentation.")
```
Comment (juliusv) [inline main.go]: This really needs a link to a migration doc :)

Comment (fabxc) [inline main.go]: Once we have one :)

**Comment by juliusv** [comment]
Ok, :+1: for now :)

**Comment by discordianfish** [comment]
LGTM

**Pull request merged**